.brv/context-tree/development/ito_workflow/obsolete_specialist_cleanup.abstract.md

```
@@ -0,0 +1 @@
+Installer cleanup now pre-removes obsolete ito-orchestrator specialist files and broken symlinks on update or forceful init/reinstall, while preserving coordinator assets and user files on plain init.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)